### DISCUSSION TITLE
Start a Wiki on this git to get some documentation started.

### DISCUSSION THREAD

**Discussion by EddieSoloProjects** [Sixze/ALS-Refactored#321] [Ideas]
Can a Wiki be made that the public can submit to. I need some documentation and having the public help write it may get it done quicker.

**Comment by jb4twin** [comment]
I am completely up to make a quick start guide once I understand how to make it works, pass the initial step of having my own character running.
The most useful ressource would be to know how to integrate new animations as the system differ from ALS.

**Comment by jb4twin** [comment]
I would be more than happy to make a tutorial to answer to this question:
-> https://github.com/Sixze/ALS-Refactored/discussions/99

This is the very basic instructions, I'm trying to figure out.
Any help much appreciated

```
1.copy one of overlaystate Animblueprint which most same with your
2.add tag for your overlay
3.add bot of it to character overlay map value
4.replace anim in animblueprint
5. adjust anim curve
```

**Comment by jb4twin** [comment]
Another useful thread
-> https://github.com/Sixze/ALS-Refactored/discussions/19

**Comment by EddieSoloProjects** [comment]
I have found that adding Player Skeletal Meshes by deleting the skeleton and using the "Replace With" option in the delete sometimes results in twisted arms. But you can Export a FBX for a UE4 Skeletal Mesh. And when you Re-import it, Select the **_ALS Skeleton_** as an import option. Now the Skeletal mesh works perfectly with ALS. :)
